ansible/roles/nginx/files/genssl.sh

```diff
@@ -56,7 +56,7 @@ else
     openssl req -new \
     -key "$SCRIPTDIR/openwhisk-client-key.pem" \
     -passin pass:$PASSWORD \
-    -subj "/C=US/ST=NY/L=Yorktown/O=OpenWhisk/CN=guest" \
+    -subj "/C=US/ST=NY/L=Yorktown/O=OpenWhisk/CN=whisk.system" \
```

Comment:
Is guest usable without a cert? I think so but checking. If so then can we avoid changing the catalog scripts?